src/popup/pages/AccountDetails.vue

```diff
@@ -106,7 +112,11 @@ export default defineComponent({
     InputSearch,
   },
   setup(props, { root }) {
+    const accountDetails = ref<HTMLElement>();
```

Comment:
I suggest to rename this constant to something more meaningful, like `rootElem` or `accountDetailsElem`.